spec/models/institution_spec.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Institution, type: :model do
+  describe 'methods' do
+    it "has a full address method from it's components" do
+      type = create(:institution_type)
+      i = create(:institution, institution_type: type)
```

Comment:
`i` is a terrible variable name, readability is one of the most important aspects of a well written and maintainable code
```suggestion
      institution = create(:institution, institution_type: type)
```

Reply:
I know, this was the previous variable name but after rubocop screamed that the line is too long.